src/components/molecules/BalanceGameEndingBox/BalanceGameEndingBox.tsx

```diff
@@ -25,10 +26,17 @@ const BalanceGameEndingBox = ({
   isMyEndBookmark,
 }: BalanceGameEndingBoxProps) => {
   const currentURL: string = window.location.href;
+  const isGuest = !localStorage.getItem('accessToken');
 
+  const [loginModalOpen, setLoginModalOpen] = useState<boolean>(false);
   const [shareModalOpen, setShareModalOpen] = useState<boolean>(false);
   const { isVisible, modalText, showToastModal } = useToastModal();
 
+  const onHandleLogin = () => {

```

Comment:
취향 차이일 수도 있지만, 
```suggestion
  const handleLogin = () => {
```
handle 정도로 처리해도 충분해보여요! 혹은 `onLoginClick` , `onButtonClick` 정도로 표현하면 좋을 거 같아요

Reply:
[a72a887](https://github.com/JECT-Study/PICK-O-Client/pull/229/commits/a72a88738b85efb0f4eb589b7a7e276d7f564a5e) 수정했습니다!!